src/AbstractScrollbar.js

```diff
@@ -401,4 +406,28 @@ function getMouseWheelEventName() {
 	} else if ('onwheel' in document.documentElement) {
 		return 'wheel';
 	}
+}
+
+/**
+ * Apply necessary styles for proper computation of scrollbar sizes.
+ * It returns clearer function which removes all added styles from DOM.
+ * @returns {Function} clearer function, removing inlined styles from DOM
+ */
+function fixScrollStyles() {
+	let styleFixEl = document.createElement('style'),
+		styleFixText = `.${CN_SCROLLBAR__CONTAINER}::-webkit-scrollbar { display: none; }`;
```

Comment:
according to codestyle please don't use one let/var/const for multiple declarations
besides, you should have had an eslint error here and on line 432